src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import core.basesyntax.model.Color;
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Color getRandomColor() {
```

Comment:
The getRandomColor() method should not be a static
```suggestion
    public Color getRandomColor() {
```